capabilities/istio/.github/zarf-runner/deploy/zarf.yaml

```diff
@@ -22,15 +22,10 @@ components:
                 ARCH=$(uname -m)
                 [ "$ARCH" = "x86_64" ] && ARCH="amd64"
                 ./zarf p d ../build/zarf-package-uds-capability-istio-$ARCH-1.19.3.tar.zst  --no-progress --confirm #${COMPONENTS}          
-          - description: Inject tls certificate
-            cmd: |
-              npm ci
```

Comment:
I understand why we are removing this here, but just to clarify, do we still plan on using this in our actual deployments or just kubectl?

Reply:
Move to the chart in a different area, it's still ugly now until helm variable things exist.